### PR TITLE
Display Address validation

### PR DESCRIPTION
**Pull request by ezequielc**
Validates Display address when not using Company address.
[stage-1]

@alex-deaconu Please review. Thanks!

### CODE REVIEW

File: web/scripts/displays/services/svc-display-factory.js [hunk 1]
```diff
@@ -126,6 +127,14 @@ angular.module('risevision.displays.services')
         return deferred.promise;
       };
 
+      var _validateAddress = function () {
+        if (factory.display.useCompanyAddress) {
+          return $q.resolve();
+        } else {
+          return addressFactory.isValidOrEmptyAddress(factory.display);
```
Comment (alex-deaconu) [inline web/scripts/displays/services/svc-display-factory.js]: @ezequielc should we allow empty addresses for Display? To me, if the user un-checks "Use Company Address" they should probably enter an address. Just a though. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good, a comment inline. Thanks!

File: web/scripts/displays/services/svc-display-factory.js (hunk 1 above)
Comment (ezequielc) [reply web/scripts/displays/services/svc-display-factory.js]: @alex-deaconu Very good point. I will update and not allow blank address. Thanks

Comment (ezequielc) [reply web/scripts/displays/services/svc-display-factory.js]: @alex-deaconu Updated as suggested, please review the last commits. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc looks good 👍 . Thanks for updating!

**Pull request merged**